### PR TITLE
AR-542: Initial auth. Bundle and security setup

### PR DESCRIPTION
**Pull request by turegjorup**
#### Link to ticket

https://jira.itkdev.dk/browse/AR-542

#### Description

Added JWT auth to API with local users

#### Screenshot of the result

No screenshot

#### Checklist

- [x] My code is covered by test cases.
- [x] My code passes our test (all our tests).
- [x] My code passes our static analysis suite.
- [x] My code passes our continuous integration process.

If your code does not pass all the requirements on the checklist you have to add a comment explaining why this change 
should be exempt from the list.

#### Additional comments or questions

If you have any further comments or questions for the reviewer please add them here.

### CODE REVIEW

File: src/Repository/UserRepository.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        if (!$user instanceof User) {
+            throw new UnsupportedUserException(sprintf('Instances of "%s" are not supported.', \get_class($user)));
+        }
+
+        $user->setPassword($newHashedPassword);
+        $this->_em->persist($user);
+        $this->_em->flush();
+    }
+
+    // /**
```
Comment (tuj) [inline src/Repository/UserRepository.php:39]: Remove

File: .env [hunk 2]
```diff
@@ -38,3 +38,24 @@ CORS_ALLOW_ORIGIN='^https?://(localhost|127\.0\.0\.1)(:[0-9]+)?$'
 ###> App ###
 APP_DEFAULT_DATE_FORMAT='Y-m-d\TH:i:s.v\Z'
 ###< App ###
+
+###> lexik/jwt-authentication-bundle ###
+JWT_SECRET_KEY=%kernel.project_dir%/config/jwt/private.pem
+JWT_PUBLIC_KEY=%kernel.project_dir%/config/jwt/public.pem
+JWT_PASSPHRASE=4abd42d7e2392b986081f98617b66be2
```
Comment (tuj) [inline .env:45]: Should this line be blank or replaced with REPLACE_ME or something?

File: src/Repository/UserRepository.php (hunk 1 above)
Comment (turegjorup) [reply src/Repository/UserRepository.php:39]: This is an auto-generated file. So far we have chosen to leave them as is.

File: migrations/Version20211221131836.php [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<?php
```
[the remark this answers is not in the document]
Comment (turegjorup) [reply migrations/Version20211221131836.php:1]: We can roll up the migrations when we are ready for 1.0

**Review by tuj — APPROVED** [review]

**Pull request merged**